google/cloud/security/common/data_access/violation_dao.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Provides the data access object (DAO) for Organizations."""
+
+import MySQLdb
+
+from google.cloud.security.common.data_access import dao
+from google.cloud.security.common.data_access import errors as db_errors
+from google.cloud.security.common.data_access.sql_queries import load_data
+from google.cloud.security.common.util import log_util
+
+LOGGER = log_util.get_logger(__name__)
+
+
+class ViolationDao(dao.Dao):
+    """Data access object (DAO) for rule violations."""
+
+    RESOURCE_NAME = 'violations'
+
+    def import_violations(self, violations, snapshot_timestamp=None):
```

Comment:
nit: Can we change this method naming to "insert_violations", to be more aligned and descriptive of the sql operation?  It would be so much clearer when it's used elsewhere.